services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_core.py

```diff
@@ -107,6 +109,13 @@ async def create_service_and_get_id(
             create_service_data, by_alias=True, exclude_unset=True
         )
         kwargs = {_to_snake_case(k): v for k, v in kwargs.items()}
+        if docker_hub_registry_settings:
+            kwargs["auth"] = {
+                "username": docker_hub_registry_settings.REGISTRY_USER,
+                "password": docker_hub_registry_settings.REGISTRY_PW.get_secret_value(),
+                "serveraddress": docker_hub_registry_settings.resolved_registry_url,
+            }
+            kwargs["registry"] = docker_hub_registry_settings.resolved_registry_url
```

Comment:
did you notice that  `kwargs["registry"] == kwargs["auth"]["serveraddress"]`  ? 

Please double check. I di noot see the registry part in the manual